components/dropdown/test/dropdown-content.visual-diff.html

```diff
@@ -397,6 +397,62 @@ <h1>Header Larger Than Max</h1>
 			</d2l-dropdown-content>
 		</d2l-dropdown>
 
+		<d2l-dropdown id="mobile-right-tray">
```

Comment:
Added some tests for mobile screens. Looks like all the tests previously were using a 400*800 screen.